src/ken.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Kensetsu KEN | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Learn about KENSETSU, SORA’s MakerDAO equivalent, involving XOR token burn, KEN token rewards, and Kensetsu’s role in the DeFi space on Polkadot
+  - - meta
+    - name: keywords
+      content: token burn, SORA, Maker DAO, stablecoin
+---
+
+# Kensetsu KEN
+
+::: info
+This token was originally proposed in [SORA RFP 60](https://github.com/sora-xor/rfps/issues/60) by Satoshi Shibarimoto.
+:::
+
+## TL;DR
+
+- All of the initial KEN tokens will be distributed to accounts that burn at least 1 million XOR between blocks 14,464,000 and 14,939,200
+- Burning less than 1 million XOR will yield no KEN tokens
+- 1% of all Kensetsu stablecoins minted will be used to buy back and burn KEN on Polkaswap
+- 80% of burned KEN burned daily will be reminted and be given to Demeter Farm for farming rewards on the XOR-KUSD pool, to incentivize liquidity
+
+## Kensetsu Introduction
+
+Maker DAO is the largest DeFi app, with $5.92 Billion in TVL; Maker DAO powers DAI, the decentralized dollar-pegged stablecoin. SORA is a new world economic system aimed at high, sustainable growth. Despite SORA’s bold ambitions, it does not have a Maker DAO equivalent on its network. That’s why in 2023, SORA community contributor Satoshi Shibarimoto, proposed KENSETSU as SORA’s version of Maker DAO that not only builds on the successes of the original protocol but also takes advantage of the amazing Polkadot tech.
+
+With KENSETSU, XOR token holders will be able to vote to create stablecoins pegged to any values available via oracles on the SORA network, such as popular fiat currencies, GOLD, SILVER, and others. As with Maker DAO, users can mint stablecoins by locking up collateral in vaults.
+
+In KENSETSU, there will be two vault types: 1) vaults where users lock up collateral and they can get the collateral back by paying back the minted stablecoins, and 2) vaults where users lock up collateral, but to get it back, they will have to pay the minted stablecoins + a stability fee; these vaults can be liquidated if the value of the underlying collateral goes too low; type-1 vaults cannot be liquidated, but there will be a stability fee as a tax that is taken from the collateral over time, so users won’t ever get their total collateral back. Initial collateral assets are expected to be XOR, TBCD, VAL, PSWAP, ETH, and DAI.
+
+The ability to mint stablecoins from collateral is very useful. Still, because there is a stability fee on the vaults, there also a need for the the stablecoins to be liquid and create demand. That’s where KEN comes in.
```

Comment:
```suggestion
The ability to mint stablecoins from collateral is very useful. Still, because there is a stability fee on the vaults, there also is a need for the the stablecoins to be liquid and create demand. That’s where KEN comes in.
```